Python/Wikipedia_Footballer_Scraper/main.py

```diff
@@ -0,0 +1,107 @@
+from bs4 import BeautifulSoup
+import requests
+
+
+def get_player_name():
+    """
+    Takes Football player's name input from the user and
+    returns a string suitable to concatenate in the wikipedia URL.
+    """
+
+    player = input('\nEnter the name of the Football player: ').split(' ')
+    player_name = ''
+    for i in player:
+        name = i.capitalize()
+        player_name += name
+        player_name += ' '
+
```

Comment:
Also an important operation you might need to have in this function would be to handle the leading spaces in entered playernames. Eg. what if user enters `Messi     ` instead of `Messi`. I guess you get what issue will happen; so as an enhancement, this can also be a good enhancement to have in your code!

Reply:
As far as I tested the code, adding leading or preceeding spaces to the input player name generate no errors and have no effect in the URL.